docs/guide/apm-breaking-changes.asciidoc

```diff
@@ -23,6 +24,18 @@ Also see <<apm-release-notes>>.
 
 // end::notable-v8-breaking-changes[]
 
+[[breaking-7.6.0]]
+=== 7.6.0 APM Breaking changes
```

Comment:
@cauemarcondes / @ogupte  -- Any APM app breaking changes for 7.6 that should be mentioned here?